app/views/forms/edit/document-selector.js

```diff
@@ -337,7 +337,7 @@ function ($http, $rootScope, $filter, $q, searchService, appConfigService, IStor
                 $scope.addNewContact = true;
                 require(['views/forms/edit/directives/edit-contact.directive'], function(){
                     
-                    var directiveHtml = "<DIRECTIVE on-post-publish='onNewContactPublish(data)' link-target={{linkTarget}} locales='locales'></DIRECTIVE>"
+                    var directiveHtml = "<DIRECTIVE on-post-publish='onNewContactPublish(data)' is-dialog='true' container='.ngdialog' link-target={{linkTarget}} locales='locales'></DIRECTIVE>"
```

Comment:
why not just <edit-contact> instead of the .replce
```suggestion
                    var directiveHtml = "<edit-contact on-post-publish='onNewContactPublish(data)' is-dialog='true' container='.ngdialog' link-target={{linkTarget}} locales='locales'></edit-contact>"
```